dataduct/pipeline/s3_node.py

```diff
@@ -59,6 +60,11 @@ def __init__(self,
         else:
             additional_args['filePath'] = s3_object
 
+        if (compression == "gzip"):
```

Comment:
Will do - wasn't sure about python style guidelines here. 

no bzip2/lzo because it's not supported by amazon. Why it isn't available here, but is in RedshiftCopyAcitivity? no clue.